docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/configuration-reference-azure/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Lake Loader Azure configuration reference"
+date: "2023-09-18"
+sidebar_position: 2
+---
+
+
+```mdx-code-block
+import {versions} from '@site/src/componentVersions';
+import CodeBlock from '@theme/CodeBlock';
+```
+
+<p>The configuration reference in this page is written for Lake Loader <code>{`${versions.lakeLoader}`}</code></p>
```

Comment:
I think I would prefer a single configuration page with tabs. Especially because we could have two independent groups of tabs: for the cloud and for the data format.